Makefile

```
@@ -17,6 +17,19 @@ static-server:
 # Prod server - persistent background process without process manager or Docker
 # Uses sudo to bind to port 80 and/or 443
 
+# # Docker
+
+docker-build:
+	docker image build -t dopplerhq/yodaspeak:latest .
+
+# Needs `DOPPLER_TOKEN` env var to fetch secrets from Doppler's API
+# Learn more at https://docs.doppler.com/docs/enclave-service-tokens
+docker-run:
+	docker container run -it --rm -e DOPPLER_TOKEN=$(DOPPLER_TOKEN) -p 3000:3000 yodaspeak:latest
```

There's one last instance that snuck by

```suggestion
	docker container run -it --rm -e DOPPLER_TOKEN=$(DOPPLER_TOKEN) -p 3000:3000 dopplerhq/yodaspeak:latest
```